### PR TITLE
RUN-3335 Normalize Microsoft constants

### PR DESCRIPTION
**Pull request by joneit**
Also collapsed some whitespace.
[Tests](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/59c16f307cb79f732d10c36a)

### CODE REVIEW

**Comment by HarsimranSingh** [comment]
👍

**Comment by joneit** [comment]
Rebased to `8.56.25.3`. (Will rerun tests closer to merge time in case another rebase is required.)

File: src/browser/api_protocol/api_handlers/application.js [hunk 1]
```diff
@@ -350,71 +340,58 @@ function deregisterExternalWindow(identity, message, ack) {
...
             ofEvents.emit(route.externalWindow('close', uuid, name));
             break;
-        case 7:
-            // WM_SETFOCUS
+        case WindowsMessages.WM_SETFOCUS:
             ofEvents.emit(route.externalWindow('focus', uuid, name));
             break;
-        case 8:
-            // WM__KILLFOCUS
+        case WindowsMessages.WM__KILLFOCUS:
```
Comment (datamadic) [inline src/browser/api_protocol/api_handlers/application.js]: This is undefined... `WM__KILLFOCUS` vs `WM_KILLFOCUS`. How can we confirm these?

Comment (joneit) [reply src/browser/api_protocol/api_handlers/application.js]: we could do WindowMessages.get('KILLFOCUS') which could log an error (or worse) if KILLFOCUS is unknown, but would be unusual coding and probably overkill.

Comment (joneit) [reply src/browser/api_protocol/api_handlers/application.js]: I searched for `__` inside a word and the above was the only hit:
```bash
egrep -r '\w+__\w+' src
```

**Comment by wenjunche** [comment]
👍

**Pull request merged**